x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/malware.tsx

```diff
@@ -177,6 +187,23 @@ export const MalwareProtections = React.memo(() => {
       rightCorner={protectionSwitch}
     >
       {radioButtons}
+      <EuiSpacer size="m" />
+      <EuiCallOut iconType="iInCircle">
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.details.detectionRulesMessage"
+          defaultMessage="Click {detectionRulesLink} to view related detection rules. Prebuilt rules are tagged “Elastic” on the Detection Rules page."
+          values={{
+            detectionRulesLink: (
+              <LinkToApp appId={`${APP_ID}:${SecurityPageName.detections}`} appPath={`/rules`}>
```

Comment:
something to think about for the future: write the link like:

> View [related detection rules](). Prebuilt rules are tagged "Elastic" on the Detection Rules page.

This way, when assistive tech lists all the links on the page, the user will get 'related detection rules' in the list instead of 'here'. Also users eyes are drawn to links and if they see "[here]()" they have to scan around the text and read the whole sentence to know where the link goes.

https://webaccess.berkeley.edu/ask-pecan/click-here

Reply:
good suggestion, I pushed up the change